docs/wiki/ALZ-Deprecated-Services.md

```diff
@@ -21,4 +21,6 @@ Over time, a deprecation process of there `ALZ / custom` policies will have to t
 
 | Deprecated ALZ Policy IDs                     | Superseded by built-in policy IDs    | Justification                                                            |   |   |
 |-----------------------------------------------|--------------------------------------|--------------------------------------------------------------------------|---|---|
-| <ul><li>Deploy-Nsg-FlowLogs</li><li>Deploy-Nsg-FlowLogs-to-LA</li></ul> | e920df7f-9a64-4066-9b58-52684c02a091 | Custom policy replaced by built-in requires less administration overhead |   |   |
+| Deploy-Nsg-FlowLogs | e920df7f-9a64-4066-9b58-52684c02a091 | Custom policy replaced by built-in requires less administration overhead |   |   |
+| Deploy-Nsg-FlowLogs-to-LA | e920df7f-9a64-4066-9b58-52684c02a091 | Custom policy replaced by built-in requires less administration overhead |   |   |
+| Deny-PublicIP | 6c112d4e-5bc7-47ae-a041-ea2d9dccd749 | Custom policy replaced by built-in requires less administration overhead |   |   |
```

Comment:
Can we also fix this table as there appears to be some additional unused columns on the right